src/algorithms/rhythm/harmonicbpm.cpp

```diff
@@ -55,12 +48,20 @@ vector<Real> HarmonicBpm::findHarmonicBpms(const vector<Real>& bpms) {
     throw(EssentiaException("HarmonicBpm: Zero bpm value found"));
   }
 
-  // Eventhough 0 is the min. documented value you can supply
-  // It can cause Essentia to crash
+  // FIXME 
+  // Eventhough 0 is the min. documented value you can supply, 
+  // A _tolerance = 0 causes an endless loop.
+  // Prevent this scenario until long term fix found in loop below.
   if (_tolerance == 0){ 
     throw(EssentiaException("HarmonicBpm: Zero tolerance found"));
   }
 
+  // finding the minimum bpm and check it is above 20
+  int min = *min_element(bpms.begin(), bpms.end());
+  if (min < 20 ){
+    E_INFO ("HarmonicBpm: There are very low values among bpms candidates (below 20 BPM)");
+  }
+
```

Comment:
I am not sure about this threshold (=20). It is reasonable that a BPM of 20 is weirdly slow, but it is a sub-harmonic of common BPMs that can occur (40, 60, 80, etc.) and in theory it can occur in some BPM estimation algorithms. For the `bpm` parameter we require >= 1. Why not using the same?

Reply:
I split the warnings statement scenarios into:
(1) low BPM (to cover the possible sub harmonic range below 20 when threshold is also set very low
(2) extremely low BPM for value equal or below 1.